pipeline-templates/terraform.yaml

```diff
@@ -1,11 +1,29 @@
-jobs:
-  - job: Plan_${{ parameters.environment }}_${{ parameters.component }}
-    pool:
-      vmImage: ${{ parameters.agentPool }}
-    steps:
+  jobs:
+    - job: TerraformPlanAndApply
+      pool: ${{ parameters.pool }}
+      timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+      steps:     
       - task: TerraformInstaller@0
+        displayName: Terraform install
         inputs:
           terraformVersion: ${{ parameters.terraformVersion }}
+
+      - task: PowerShell@2
```

Comment:
this only needs to be run once per pipeline not on every terraform run.

Reply:
this is taking 15-20 seconds on each job right now

Reply:
Added to the current "validate" stage